### ISSUE TITLE
Any integration of p or m class system?

### ISSUE THREAD

**Issue by Nojil** [jgthms/bulma#1890]
This is about **Bulma **.

This is a feature request/question.

### Overview of the problem

This is about the Bulma **CSS framework**

### Description
With bootstrap there is the class system to use p-* or m-* to edit the padding and margin of a container. For example you can use pl-0 (padding-left = 0).

Does this exist with Bulma? If not it would be nice if it was added, its the only thing keeping me from switching over form bootstrap 4.

**Comment by Nojil** [comment]
Yes when adding the class of p-0 it removes the padding from the element same goes for m-0, where p-5 adds padding around the entire element.

Both p- and m- act the same however one modifies pudding and the other margin. You can also use pr- and pl- for padding right and padding left along with pt- and pb- for padding top and padding bottom. You can also use px- to modify the padding left and right and py- to modify the padding top and bottom. You can do the exact same with margin using m in replace of p

**Comment by jgthms** [comment]
You can already use `.is-marginless` and `.is-paddingless`. We could add the same for each of the 4 sides.

However, having `px-` is like designing in the HTML, which I want to avoid. I feel like people would just use these classes all over the places. I've seen terrible things with such a system.

**Comment by Nojil** [comment]
Here is more information on the subject from the bootstrap docs.

[Bootstrap Documentation](https://getbootstrap.com/docs/4.1/utilities/spacing/)
